scripts/test/launchTestUtilities.ts

```diff
@@ -114,67 +196,11 @@ export async function getCliArgsToDisableExtensions(
     return ids
 }
 
-export function getMinVsCodeVersion(): string {
+function getMinVsCodeVersion(): string {
     const vsCodeVersion = packageJson.engines.vscode
 
     // We assume that we specify a minium, so it matches ^<number>, so remove ^'s
     const sanitizedVersion = vsCodeVersion.replace('^', '')
     console.log(`Using minimum VSCode version specified in package.json: ${sanitizedVersion}`)
     return sanitizedVersion
```

Comment:
```suggestion
    const sanitizedVersion = getMinVscodeVersion()
    console.log(`Using minimum VSCode version specified in package.json: ${sanitizedVersion}`)
    return sanitizedVersion
```

Reply:
I was assuming that we do not want to import our extension code in to the scripts. But if there isn't a reason not to then this makes sense.

Reply:
There is no reason not to. Except code that requires Toolkit init/activation, perhaps.